frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/EndHearing.js

```diff
@@ -1,18 +1,124 @@
-import React from "react";
+import { CardText, Modal, CloseSvg } from "@egovernments/digit-ui-react-components";
+import React, { useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useHistory } from "react-router-dom";
+import SummaryModal from "../../components/SummaryModal";
 
+const fieldStyle = { marginRight: 0 };
 
-const fieldStyle={ marginRight: 0 };
+const Heading = (props) => {
+  return (
+    <div style={{ width: "440px", height: "56px" }}>
+      <p
+        style={{
+          fontWeight: 700,
+          fontSize: "24px",
+          lineHeight: "28.13px",
+          color: "#0A0A0A",
+        }}
+      >
+        {props.label}
+      </p>
+    </div>
+  );
+};
+
+const Close = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#0A0A0A">
+    <path d="M0 0h24v24H0V0z" fill="none" />
+    <path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z" />
+  </svg>
+);
+
+const CloseBtn = (props) => {
+  return (
+    <div onClick={props?.onClick} style={{ paddingTop: 10 }}>
+      <div className={"icon-bg-secondary"} style={{ backgroundColor: "#ffff", cursor: "pointer" }}>
+        {" "}
+        <Close />{" "}
+      </div>
+    </div>
+  );
+};
+
+const EndHearing = ({ handleEndHearingModal, hearingId, hearing }) => {
+  const { t } = useTranslation();
+  const tenantId = window?.Digit.ULBService.getCurrentTenantId();
+  const [openConfirmationModal, setOpenConfirmationModal] = useState(false);
+
+  const { data: hearingResponse, refetch } = Digit.Hooks.hearings.useUpdateHearingsService(
```

Comment:
mentioned this to maruthi as well, remove useUpdateHearings, just call hearingService.updateHearing directly where needed